scripts/core/common.yml

```diff
@@ -316,3 +316,12 @@ members:
     - type: uint64_t
       name: depth
       desc: "[in] scalar (scalar)"
+--- #--------------------------------------------------------------------------
+type: typedef
+desc: "File Descriptor type."
+name: $x_file_descriptor_t
+value:
+    - if: Windows
+      value: HANDLE
+    - else: ""
```

Comment:
I don't think the `""` is strictly necessary from a yaml spec point of view, were the scripts complaining about it not having a value?

Reply:
ah, I see, the value of `else:` is `None` so the checks are not doing the correct thing, I'll make some suggestions.

Reply:
```suggestion
    - else:
```